packages/data/src/onboarding/reducer.js

```diff
@@ -6,7 +6,21 @@ import TYPES from './action-types';
 const onboarding = (
 	state = {
 		errors: {},
-		profileItems: {},
+		profileItems: {
+			business_extensions: null,
```

Comment:
I really think we should be doing this default state stuff everywhere. Its informative to the next person and it also makes the type of the reducer consistent. I just need to fix up the tests to account for this.